CenterForOpenScience/osf.io
# [ENG-2306] Four new registration forms

## Purpose

Adds new schemas for OSF preregistration

## Changes

- 4 new schemas and a migrations

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify all language is the same as as the excel spreadsheet
- Verify all schemas are initially invisible

## Documentation

simple tech task, no docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2306
